venia-integration-tests/README.md

```diff
@@ -7,6 +7,28 @@
 3. Run `yarn test`
 4. Now select test to Run from Cypress UI.
 
+## Run tests in headless mode
+
+Running tests in the headless mode is the preferred option when testing in the CI. Before updating/creating the tests please run the tests in the headless mode to make sure they are running as expected.
```

Comment:
did you mean "Command Line" instead of "CI", i.e. Continuous Integration?

```suggestion
Running tests in the headless mode is the preferred option when testing in the CI. Before updating/creating any tests, please run the test suite in headless mode to make sure they are running as expected.
```

Reply:
Continuous Integration